site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
+* Disposition Frames:
+  * Accepted: Message successfully processed by the adapter.
+  * Released: Message cannot be processed and should be redelivered.
+  * Rejected: Adapter rejects the message due to one of the following:
+        * (`hono:bad-request`): Request rejected due to a bad client request.
+        * (`amqp:unauthorized-access`): Request rejected because the adapter is disabled for tenant.
+        * (`amqp:precondition-failed`): Request does not fulfill certain requirements e.g adapter cannot assert device registration etc.
+
+When a device publishes data to the `telemetry` address, the AMQP adapter automatically determines the device's identity and tenant during the authentication process.
+
+**Example**
+
+Publish some JSON data for device `4711`:
+
+    $ qpid-send -b "localhost:4040" --connection-options "{username:sensor1@DEFAULT_TENANT, password:hono-secret, protocol:amqp1.0}" -a "'telemetry'" --content-string '{"temp": 5}'
+
+We supply the device credentials, as well as the AMQP protocol version, in the connection options of the *qpid-send* command. The presence of the username and password options indicates that we want to authenticate using SASL PLAIN mechanism.
+
+## Publish Telemetry Data (unauthenticated Device)
+
+* Address: `telemetry/${tenant-id}/${device-id}` or `t/${tenant-id}/${device-id}`
+* Delivery: `AT_MOST_ONCE` or `AT_LEAST_ONCE`
+* Authentication: none
+* Message Header(s):
+  * (optional) Arbitrary headers (durable, ttl, priority, ...)
+* Message Body:
+  * (optional) Arbitrary payload
+* Message properties:
+  * (optional) Arbitrary properties (content-type, correlation-id, ...)
+* Disposition Frames:
+  * Accepted: Message successfully processed by the adapter.
+  * Released: Message cannot be processed and should be redelivered.
+  * Rejected: Adapter rejects the message due to:
+        * (`hono:bad-request`): A bad client request (e.g invalid content-type).
+        * (`amqp:unauthorized-access`): The adapter is disabled for tenant.
+        * (`amqp:precondition-failed`): Request not fulfilling certain requirements.
+
+Note how verbose the address is for unauthenticated devices. This address can be used by devices that have not authenticated to the protocol adapter. This requires the `HONO_AMQP_AUTHENTICATION_REQUIRED` configuration property to be explicitly set to `false` before starting the protocol adapter.
+
+**Examples**
+
+Publish some JSON data for device `4711`:
+
+    $ qpid-send -b "localhost:4040" --connection-options "{protocol:amqp1.0}" -a "'telemetry/DEFAULT_TENANT/4711'" --content-string '{"temp": 5}'
+
+Note the address format and the absence of the username/password property for the case of unauthenticated devices.
+
+## Publishing Events
+
+The AMQP adapter supports publishing of events to Hono's Event API. Events are published using only `AT_LEAST_ONCE` delivery semantics.
+
+## Publish an Event (authenticated Device)
+
+* Address: `event` or `e`
```

Comment:
really? is `e` supported? is `t` also supported for `telemetry`?

Reply:
yes, the telemetry/event case considers both.. see `EndpointType#fromString`